shared/macros.jinja

```diff
@@ -15,6 +15,30 @@ ocil_clause: "the required value is not set"
 {{%- endmacro %}}
 
 
+{{% macro local_users_considered_only() -%}}
+This rule only considers local users.
+If you have your users defined outside <code>/etc/passwd</code>, the rule won't consider those.
```

Comment:
Lines 21-26 aren't need and are very wordy. 19-20 are really good and sufficient. Drop lines 21-26.